spec/models/authenticator/amazon_spec.rb

```diff
@@ -127,7 +127,11 @@ class FakeGroup < Struct.new(:name)
     allow_any_instance_of(described_class).to receive(:aws_connect) { |*args| FakeAmazon.new(user_data, *args) }
   end
 
-  its(:uses_stored_password?) { should be_false }
```

Comment:
Heh. Yeah, just refers to `subject`. Removed in RSpec 3 to a gem: https://github.com/rspec/rspec-its  Reasoning: https://gist.github.com/myronmarston/4503509